components/support/migration/src/main/java/mozilla/components/support/migration/FennecFxaMigration.kt

```diff
@@ -191,16 +191,22 @@ private object AuthenticatedAccountProcessor {
 internal object FennecFxaMigration {
     private val logger = Logger("FennecFxaMigration")
 
-    private const val DEFAULT_TOKEN_SERVER_URL = "https://token.services.mozilla.com/1.0/sync/1.5"
-    private const val DEFAULT_IDP_SERVER_URL = "https://api.accounts.firefox.com/v1"
+    // From https://accounts.firefox.com/.well-known/fxa-client-configuration
+    private const val GLOBAL_TOKEN_SERVER_URL = "https://token.services.mozilla.com/1.0/sync/1.5"
+    private const val GLOBAL_IDP_SERVER_URL = "https://api.accounts.firefox.com/v1"
+
+    // From https://accounts.firefox.com.cn/.well-known/fxa-client-configuration
+    private const val CHINA_TOKEN_SERVER_URL = "https://sync.firefox.com.cn/token/1.0/sync/1.5"
+    private const val CHINA_IDP_SERVER_URL = "https://api-accounts.firefox.com.cn/v1"
```

Comment:
nit: In Fenix Stefan suggested to use "Mozilla Online" instead of "china" as description and that's what we have been using for the build flags there (`-PmozillaOnline`)